test/integration/happy_path_test.rb

```diff
@@ -1,7 +1,21 @@
 class HappyPathTest < ActionDispatch::IntegrationTest
 
   test 'shows home page' do
-    visit root_path 
-    assert page.has_content?('Hello World')
+    visit root_path
+    assert page.has_content?('JetFuel')
+    fill_in "url[original_url]", :with => 'https://github.com/larsonkonr/dinner_dash/blob/master/Gemfile'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.